### PR TITLE
came_case functions for some direct classes

### PR DESCRIPTION
**Pull request by wezu**
The file changes have been made with an automated tool and then hand checked - I didn't find error but I'm also dyslexic.

### CODE REVIEW

**Comment by darktohka** [comment]
Some of these functions are only available under __debug__, so importing these files without __debug__ is not possible.

**Comment by Moguri** [comment]
Have you actually tested some of these changes? I think you're missing "self." in most (all?) of your assignments.

**Comment by rdb** [comment]
These are at the class level, so use of `self.` would not be appropriate.
Adding these to the instance dict would be unnecessarily inefficient.

However, it would be great if the functions that are defined conditionally
on `__dev__` could be fixed, since this is breaking (as Travis indicates)

**Comment by Moguri** [comment]
Ah, I didn't notice they are at the class level.

File: direct/src/actor/Actor.py [hunk 1]
```diff
@@ -2549,3 +2557,92 @@ def renamePartBundles(self, partName, newBundleName):
...
+    get_overlapping_joints = getOverlappingJoints
+    enable_blend = enableBlend
+    face_towards_viewer = faceTowardsViewer
+    bind_anim = bindAnim
+    set_blend = setBlend
+    get_frame_time = getFrameTime
+    remove_node = removeNode
+    wait_pending = waitPending
+    expose_joint = exposeJoint
+    set_lodnode = setLODNode
```
Comment (rdb) [inline direct/src/actor/Actor.py]: Should be set_lod_node

File: direct/src/actor/Actor.py [hunk 2]
```diff
@@ -2549,3 +2557,92 @@ def renamePartBundles(self, partName, newBundleName):
...
+    fix_bounds = fixBounds
+    get_anim_filename = getAnimFilename
+    get_subparts_complete = getSubpartsComplete
+    verify_subparts_complete = verifySubpartsComplete
+    get_play_rate = getPlayRate
+    clear_python_data = clearPythonData
+    load_anims = loadAnims
+    set_subparts_complete = setSubpartsComplete
+    draw_in_front = drawInFront
+    get_lodnode = getLODNode
```
Comment (rdb) [inline direct/src/actor/Actor.py]: Should be get_lod_node

**Comment by rdb** [comment]
Thanks!

**Pull request closed without merging**